extension/manifest.json

```diff
@@ -10,7 +10,8 @@
 	},
 	"permissions": [
 		"alarms",
-		"activeTab"
+		"activeTab",
+		"tabs"
```

Comment:
Looks like this isn't needed: https://developer.chrome.com/extensions/tabs#manifest